packages/flutter_tools/lib/src/flutter_project_metadata.dart

```diff
@@ -75,19 +75,19 @@ class FlutterProjectMetadata {
   FlutterProjectMetadata(File file, Logger logger) : _metadataFile = file,
                                                      _logger = logger,
                                                      migrateConfig = MigrateConfig() {
-    if (!_metadataFile.existsSync()) {
-      _logger.printTrace('No .metadata file found at ${_metadataFile.path}.');
+    if (!_metadataFile!.existsSync()) {
```

Comment:
it seems like `_metadataFile` can never be null, can you just make its type non-nullable?